package.json

```diff
@@ -74,22 +76,34 @@
     "ignore": [],
     "delay": "2500"
   },
-  "jest": {
-    "moduleFileExtensions": [
-      "ts",
-      "js"
-    ],
-    "transform": {
-      "\\.ts$": "ts-jest"
-    },
-    "moduleDirectories": [
-      "node_modules",
-      "lib"
-    ],
-    "testEnvironmentOptions": {
-      "url": "http://localhost/"
-    },
-    "testRegex": "\\.spec\\.ts$"
+  "vitest": {
+    "test": {
+      "globals": true,
+      "include": [
+        "**/*.spec.ts"
+      ],
```

Comment:
Thinking about this, we should actually exclude the `spec` files from being compiled at all in `tsconfig.json` so they don't get included in the NPM package. @paullewallencom Can you have a look into this please or want us to do this?

Reply:
they are already excluded because we have a `_spec` pattern in .npmignore